### DISCUSSION TITLE
keep sidebar open and disable mask

### DISCUSSION THREAD

**Discussion by oganator** [primefaces/community#381] [PrimeReact] [answered]
I would like to use a sidebar to hold navigation options, but also to have it kept open and to remove the mask so that a user can access the content in the body. An example can be found [here](https://mui.com/material-ui/react-drawer/) - scroll down a bit to see the 'persistent drawer'.

I find primereact much easier to use and prefer it over mui, which is why I would like to see if I can get the sidebar to work in this way. Does anyone know if this can be done on my end (for example via css class override)?

**Comment by oganator** [comment, marked as answer]
So after playing with it a bit, I found that setting dismissable={false} and modal={false} gets me most of the way there. The only thing left is to change the width of the body (outside of the sidebar) based on whether or not it is open. Is there a simple way to do this?

**Reply by melloware** [reply]
You can control the Sidebar width but you are trying to control the body width?  Why not just set them when you know the Sidebar is being opened?
